server/index.js

```diff
@@ -0,0 +1,13 @@
+require('dotenv/config');
+const express = require('express');
+const app = express();
+const path = require('path');
+const staticPath = path.join(__dirname, 'public');
+const port = process.env.PORT;
+
+app.use(express.static(staticPath));
+app.use(express.json());
+
+app.get('/', (req, res) => res.render('main'));
```

Comment:
With no routes other than serving a react app, consider instead pre-building the react app and have no backend at all.